generator/generator_tests/regions_tests.cpp

```diff
@@ -16,21 +21,48 @@
 #include <algorithm>
 #include <cstdint>
 #include <limits>
+#include <list>
 #include <memory>
 #include <string>
 #include <vector>
 #include <utility>
 
+using namespace feature;
+using namespace ftype;
+using namespace generator;
 using namespace generator_tests;
 using namespace generator::regions;
 using namespace base;
+using namespace std;
 
 namespace
 {
-using Tags = std::vector<std::pair<std::string, std::string>>;
+using Tags = vector<pair<string, string>>;
+auto NodeEntry = OsmElement::EntityType::Node;
+
+struct TagValue
+{
+  string key;
+  string value;
+
+  TagValue operator = (string const & value)
+  { return {key, value}; }
+};
+
+TagValue admin{"admin_level"};
+TagValue place{"place"};
+TagValue name{"name"};
+TagValue ba{"boundary", "administrative"};
 
-OsmElement MakeOsmElement(uint64_t id, std::string const & adminLevel,
-                          std::string const & place = "")
+struct OsmElementData
+{
+  uint64_t id;
+  vector<TagValue> tags;
+  vector<m2::PointD> polygon;
+  vector<OsmElement::Member> members;
```

Comment:
Struct member names should start with "m_".

Reply:
done